src/file.js

```diff
@@ -2071,22 +2120,30 @@ File.prototype.request = function(reqOpts, callback) {
 };
 
 /**
- * The Storage API allows you to use a custom key for server-side encryption.
  * This method allows you to update the encryption key associated with this
  * file.
  *
  * @see [Customer-supplied Encryption Keys]{@link https://cloud.google.com/storage/docs/encryption#customer-supplied}
  *
- * @param {string|buffer} encryptionKey An AES-256 encryption key.
+ * @param {string|buffer|object} options An AES-256 encryption key. If you'd
+ *     like to use a Cloud KMS key name, you must specify an options object with
+ *     the property name: `kmsKeyName`.
+ * @param {string|buffer} [options.encryptionKey] An AES-256 encryption key.
+ * @param {boolean} [options.kmsKeyName] A Cloud KMS key name.
  * @returns {File}
  *
  * @example <caption>include:samples/encryption.js</caption>
  * region_tag:storage_rotate_encryption_key
  * Example of rotating the encryption key for this file:
  */
-File.prototype.rotateEncryptionKey = function(encryptionKey, callback) {
-  var newFile = this.bucket.file(this.id, {encryptionKey});
+File.prototype.rotateEncryptionKey = function(options, callback) {
```

Comment:
does this feel idiomatic to you @stephenplusplus, it was a suggestion.

Reply:
Unfortunately, I think we're in a lose-lose situation. I think what you suggested is better than having to provide `{ kms: true }`. Having to do two things increases the odds of forgetting.

Have other languages found a better pattern when they implemented this?

Reply:
@ace-n or @jmdobry, could you help provide feedback on the surface change? I'm not a Node.js community expert and want to make sure.

@stephenplusplus, Ruby had a similar helper method, but added additional parameters for [kms key](https://github.com/GoogleCloudPlatform/google-cloud-ruby/pull/2059/files#diff-8c864fd34aa2f2f960b3b13353152551R923). However, I don't think this is how the Node.js client is meant to be used given most functions have an `options` parameter for this kind of metadata.